server/Cargo.toml

```diff
@@ -36,7 +36,7 @@ prost-types             = "0.11"
 rand                    = { version = "0.8" }
 serde                   = "1.0"
 serde_json              = "1.0"
-svc-storage-client-grpc = { git = "https://github.com/Arrow-air/svc-storage", tag = "v0.2.0" }
+svc-storage-client-grpc = { git = "https://github.com/Arrow-air/svc-storage", branch = "develop" }
```

Comment:
I think it's actually better to use the specific tags, for example there was a big merge into svc-storage just now that will break this PR if you depend on develop

Reply:
[92165fa](https://github.com/Arrow-air/svc-assets/pull/7/commits/92165faff0a4ae8b2c5d9e09519ab604d31d2a10)